### PR TITLE
Create DaSL product pages + PROOF

### PR DESCRIPTION
**Pull request by vortexing**
This PR will add the product page for PROOF along with a user guide and the new overview page for all future Tools and Platforms.

### CODE REVIEW

File: _datascience/proof.md [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+
+## What is a WDL?
+
+The **Workflow Description Language** (**WDL**, pronounced '**widdle**') is a versatile tool for executing bioinformatics and computational workflows. WDL offers:
+
+-   A **standardized** approach to describing computational tasks, including inputs, outputs, and dependencies.
+  
+-   A task-oriented approach promoting **code reuse** and modularity.
+  
+-   Support for **parallel execution**, enabling efficient processing of large datasets.
```
Comment (laderast) [inline _datascience/proof.md:31]: I would combine this with scatter-gather, because they map to the same concept

File: _datascience/proof.md [hunk 2]
```diff
@@ -0,0 +1,372 @@
...
+  
+-   Built-in support for specifying **data locations**, facilitating seamless collaboration across environments.
+  
+-   Support for **scatter-gather** operations, improving workflow performance.
+  
+-   **Flexible syntax** for concise and readable workflow definitions.
+  
+-   **Platform independence**, allows WDL workflows to run across different environments.
+  
+-   The requirement of an **engine like Cromwell** for interpreting and executing WDL on various backends enhances usability and versatility.
```
Comment (laderast) [inline _datascience/proof.md:43]: This seems more like a *how WDL works* than a *what*. Maybe this should move into the Cromwell section.

File: _datascience/proof.md [hunk 3]
```diff
@@ -0,0 +1,372 @@
...
+
+- Allows researchers to validate, troubleshoot, assess performance, and run their workflows all under one roof.
+
+- Also functions as a 'proofing' resource, enabling researchers to refine their workflows before potential transitions to cloud-based infrastructures.
+
+This guide is intended to describe how you can run PROOF, catering to varying levels of computational expertise. 
+
+We also have a WDL 101 course that will be a quick-start guide to building a WDL workflow.
+
+## What is a WDL?
```
Comment (laderast) [inline _datascience/proof.md:23]: I think this section is good, but maybe some of the bullet points should be combined to make the list shorter.

**Review by tefirman — APPROVED** [review]
Amazing work everybody! Excited to see PROOF in action! :)

**Pull request merged**